sql/src/test/java/io/crate/integrationtests/TableCompatibilitySysChecksTest.java

```diff
@@ -51,6 +53,23 @@ public void shutdown() throws IOException {
     }
 
     @Test
+    public void testRecreationRequired() throws Exception {
+        startUpNodeWithDataDir("/indices/data_home/cratedata_recreation_required.zip");
+        //set license.ident setting to avoid unlicensed cluster check
+        execute("set global transient 'license.ident' to 'my-key'");
+        execute("select * from sys.shards");
+        execute("select * from sys.checks where passed = false");
+        assertThat(response.rowCount(), is(1L));
+        assertThat(response.rows()[0][1], is(TablesNeedRecreationSysCheck.ID));
+        assertThat(response.rows()[0][3], is(SysCheck.Severity.MEDIUM.value()));
+        assertThat(response.rows()[0][0],
+            is(TablesNeedRecreationSysCheck.DESCRIPTION +
+               "[.blob_test_blob_upgrade_required, doc.test_upgrade_required, doc.test_upgrade_required_parted] " +
+               LINK_PATTERN + TablesNeedRecreationSysCheck.ID));
+    }
+
+    @Test
+    @Ignore // TODO: Re-enable after upgrade to ES 6
```

Comment:
Is it possible to change this in a way that the test passes now, but would fail after the upgrade?

Reply:
I don't think so... TableNeedsUpgrade "catches" tables with Lucene major version < Current.
This means Lucene < 6.x.x, but now if Lucene < 6.x.x then -> TablesNeedsRecreation.